src/dashboard/main.py

```diff
@@ -6,12 +6,14 @@
 
 app = Dash(__name__, use_pages=True, external_scripts=external_scripts)
 
+PORT = 3030
+
 
 def page_container() -> Component:
-    return html.Div([html.H1("PUM13-2023"), dash.page_container])
+    return html.Div(children=[dash.page_container])
```

Comment:
Går det att lägga navbar en page containern iställlet?
Vet inte om det finns något bra sätt att ta in namnet på nuvarande sidan, men om det går så skulle det bli snyggare så.

Reply:
Fixat!